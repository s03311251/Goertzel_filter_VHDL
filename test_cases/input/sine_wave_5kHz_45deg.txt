36A0
3753
3800
38A7
3948
39E3
3A77
3B04
3B8A
3C0A
3C82
3CF3
3D5D
3DC0
3E1B
3E6E
3EBA
3EFE
3F3A
3F6E
3F9A
3FBE
3FDB
3FEF
3FFB
3FFF
3FFB
3FEF
3FDB
3FBE
3F9A
3F6E
3F3A
3EFE
3EBA
3E6E
3E1B
3DC0
3D5D
3CF3
3C82
3C0A
3B8A
3B04
3A77
39E3
3948
38A7
3800
3753
36A0
35E7
3529
3465
339C
32CE
31FC
3125
3049
2F6A
2E86
2D9F
2CB5
2BC7
2AD6
29E3
28ED
27F5
26FA
25FE
2501
2402
2302
2202
2101
2000
1EFE
1DFD
1CFD
1BFD
1AFE
1A01
1905
180A
1712
161C
1529
1438
134A
1260
1179
1095
0FB6
0EDA
0E03
0D31
0C63
0B9A
0AD6
0A18
