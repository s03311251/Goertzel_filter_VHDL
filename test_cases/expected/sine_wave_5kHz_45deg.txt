FFFE2E40
FFFE3CA0
